rest_framework_bulk/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '0.2'
+__version__ = '0.2.1'
```

Comment:
hmm. wonder how this works since its already at 0.2.1 at https://github.com/miki725/django-rest-framework-bulk/blob/master/rest_framework_bulk/__init__.py#L1. am I missing anything?